llvm/include/llvm/ExecutionEngine/JITLink/JITLink.h

```diff
@@ -434,27 +437,45 @@ class Symbol {
     setTargetFlags(TargetFlagsType{});
   }
 
+  static Symbol &constructCommon(BumpPtrAllocator &Allocator, Block &Base,
```

Comment:
looks like i added this back in. Ill delete it just let the build bots currently running finish to see if there are any other issues.